header.js

```diff
@@ -0,0 +1,27 @@
+export const header = `
+<header>
+<div class="menu"><i class="fas fa-bars"></i></div>
+<nav id="primary_nav">
+<ul>
+<div class ="droit">
+<li><img src="logo.png" class ="image"></li>
+<li><a href="#" >Home</a></li>
+<li><a href="#" >Destination</a></li>
+</div>
+<div class="gauche">
+<li><a href="#"class="gauche">About us</a></li>
+
+  <li><a href="#" class="gauche">Contact us</a></li>
+  
+  <li class="deroulant" class="gauche"><a href="#">EN</a>
+    <ul class="sous">
+      <li><a href="#">FR</a></li>
+      
```

Comment:
For the selection of the language, a select (with specific style to go with the rest of the header) wouldn't be simplier ?

Reply:
Thank you Alice for all your comments and I will try to correct